wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WireExtension.kt

```diff
@@ -291,11 +293,25 @@ open class WireExtension(
     /** Calling this will resolve the configuration. */
     internal val roots: Set<File>
       get() = configuration.files + sourceDirectoriesAndLocalJars
+    private val files: ConfigurableFileCollection by lazy(NONE) {
+      val files = project.files()
+      project.dependencies.add(configuration.name, files)
+      files
+    }
 
     internal val includes = mutableListOf<String>()
     internal val excludes = mutableListOf<String>()
 
+    /**
+     * Adds a set of source. The given paths are evaluated as per [Project.files][org.gradle.api.Project.files].
+     */
+    fun srcDir(fileCollection: Any) {
+      isEmpty = false
+      files.from(fileCollection)
+    }
+
     /** Sets a directory. Example: "src/main/proto". */
+    @Deprecated("Deprecated in favor of 'srcDir(fileCollection)'")
```

Comment:
If you are deprecating this function, seems like the one below it should go as well.